components/rmrk/Gallery/PaginatedCardList.vue

```diff
@@ -136,6 +136,9 @@ export default class PaginatedCardList extends mixins(
 
   created() {
     this.fetchPageData(this.startPage)
+    if (this.client === 'bsx') {
+      this.searchQuery.sortBy = this.remapSortBy
+    }
```

Comment:
1. please check if `this.urlPrefix` is available and use it instead

2. we should add a `@Watch` for when user changes client.

Reply:
for 2. maybe we don't need to. Since after the user change client, it would redirect to homepage.

Reply:
gotcha!